Automattic/jetpack
# Google Calendar: Add google calendar block fixtures and tests

#### Changes proposed in this Pull Request:
* Refactors Google Calendar block to functional component
* Add fixtures and block tests for Google Calendar block

#### Does this pull request change what data or activity we track or use?
No change.

#### Testing instructions:
1. Create a new post, adding a Google Calendar block to it including embed url. Save post.
2. Checkout this PR, reload editor ensuring block displays correctly in editor and frontend and works as expected
3. Run tests for the block `cd projects/plugins/jetpack && yarn jest extensions/blocks/google-calendar/test/`
